src/lucky_migrator/column_default_helpers.cr

```diff
@@ -1,28 +1,36 @@
 module LuckyMigrator::ColumnDefaultHelpers
   alias ColumnDefaultType = String | Time | Int32 | Int64 | Float32 | Float64 | Bool | Symbol
 
+  def value_to_string(value : String | Time)
+    "'#{value}'"
+  end
+
+  def value_to_string(value : Int32 | Int64 | Float | Bool)
```

Comment:
I think we also need a test case and method for handling `Symbol` for `fill_existing_with: :now`

Reply:
It might also be worth passing the column type as the first argument so that the `:now` symbol only works for `Time` columns

Reply:
Yeah I see what you mean. I made the changes